### PR TITLE
Fil input v2 dialog fix

### PR DESCRIPTION
**Pull request by rajatofficial**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [ ] I ran all tests locally and they pass.

### CODE REVIEW

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml [hunk 1]
```diff
@@ -79,8 +79,7 @@
                     </basic>
                     <validation
                         jcr:primaryType="nt:unstructured"
-                        sling:resourceType="granite/ui/components/coral/foundation/include"
-                        path="core/fd/components/form/fileinput/v1/fileinput/cq:dialog/content/items/tabs/items/validation"/>
+                        sling:resourceType="granite/ui/components/coral/foundation/container"/>
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml:82]: You need to re-use the dialog from super resource type, wasn't the earlier change correct ? Can you also write a test case for this in the change.

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml (hunk 1 above)
Comment (rajatofficial) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/fileinput/v2/fileinput/_cq_dialog/.content.xml:82]: Earlier change was also correct but somehow Validation tab text was not coming hence took the reference of parent resource only. Added the test case of the Validation as well.

File: ui.tests/test-module/specs/fileinput/fileinput.authoring.spec.js [hunk 2]
```diff
@@ -100,7 +101,7 @@ describe('Page - Authoring', function () {
       cy.deleteComponentByPath(fileInputDrop);
     });
 
-    it('open edit dialog of aem forms FileInput', function() {
+    it.only('open edit dialog of aem forms FileInput', function() {
```
Comment (rismehta) [inline ui.tests/test-module/specs/fileinput/fileinput.authoring.spec.js:104]: Please remove only from the test case.

**Review by rismehta — CHANGES_REQUESTED** [review]
check comments

**Pull request merged**